lms/templates/help_modal.html

```diff
@@ -73,9 +73,9 @@
     <hr>
 
     <div class="help-buttons">
-      <a href="#" id="feedback_link_problem">${_('Report a problem')}</a>
-      <a href="#" id="feedback_link_suggestion">${_('Make a suggestion')}</a>
-      <a href="#" id="feedback_link_question">${_('Ask a question')}</a>
+      <button type="button" id="feedback_link_problem">${_('Report a problem')}</button>
```

Comment:
Have you verified that these look OK? The help menu does not seem to be enabled on your sandbox. It looks like this on staging (the button opens the modal):

![image](https://cloud.githubusercontent.com/assets/484484/17671023/b5cff0b6-62e2-11e6-9cad-61ed21e961b4.png)